snuba/query/processors/tagsmap.py

```diff
@@ -108,9 +108,8 @@ def __has_tags(self, expression: Optional[Expression]) -> bool:
         if not expression:
             return False
         for node in expression:
-            if isinstance(node, Column):
-                tag = NESTED_COL_EXPR_RE.match(node.column_name)
-                if tag and tag[1] == self.__nested_col:
+            if isinstance(node, SubscriptableReference):
+                if node.subscriptable_column.column_name == self.__nested_col:
```

Comment:
well, this is a storage processor, Once translation happens, this won't have such an easy access to the tag expression because it will be already resolved into `valueOf(indexOf(....))` by the time we get here

Reply:
I forgot about that already. :( Oh well, wishful thinking.

Probably should comment that this is a temporary nicety then.